seed/seed.json

```diff
@@ -216,14 +268,14 @@
             "filter": {
                 "min_version": "91.1.26.37",
                 "channel": ["NIGHTLY"],
-                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
+                "platform": ["ANDROID"]
             }
         },
         {
             "name": "BraveAds.CustomAdNotificationPositionedBottomRightOffsetToTheLeftOfSystemNotificationsOnWindowsStudy",
             "experiments": [
                 {
-                    "name": "AdNotifications",
+                    "name": "Enabled",
                     "probability_weight": 50,
```

Comment:
this group won't be 50% of `BraveAds.ShowCustomAdNotificationStudy::Enabled` windows users, but less as clients sign up for this study independently from the custom notification study. Maybe @goodov or @iefremov can confirm. To fix this and for better interpretability, would it make sense to re-organise all ads notifications studies by platform, i.e. have one `BraveAds.ShowCustomAdNotificationStudy` for each of the following filters:

1. Windows on Nightly and Beta: 30% custom/position-1, 30% custom/position-2, 40% default
2. Mac on Nightly and Beta: 30% custom/position-1, 30% custom/position-2, 40% default
3. Linux on Nightly and Beta: 60% custom, 40% default
4. Android on Nightly: 60% custom, 40% default

as mentioned above, we don't need the "do not show" studies as this is the default in https://github.com/brave/brave-core/blob/24fe16e3e30a7e75cf81f16bcbaa2d2982af17f0/components/brave_ads/browser/features.cc#L20

Reply:
This study will only affect users who have custom ad notifications enabled and will mean 50% of users ### will see custom ad notifications bottom right offset to the left of system notifications and the other 50% will see custom ad notifications top right (default).

Reply:
With regards to the "do not show" studies, we will need these studies for backwards compatibility before the change in https://github.com/brave/brave-core/blob/24fe16e3e30a7e75cf81f16bcbaa2d2982af17f0/components/brave_ads/browser/features.cc#L20

Reply:
I do not see how the above filters would work as we need "min_version" and "max_version" for backwards compatibility.